api/src/pcapi/domain/admin_emails.py

```diff
@@ -66,7 +66,7 @@ def send_offer_validation_notification_to_administration(
         return send_offer_creation_notification_to_administration(offer)
     if validation_status is OfferValidationStatus.REJECTED:
         return send_offer_rejection_notification_to_administration(offer)
-    return False
+    return True
```

Comment:
Pourquoi passer de `False` à `True` ? Même si j'ai l'impression qu'on ne se sert jamais de la valeur de retour de cette fonction.

Reply:
J'ai pris le parti de renvoyait "False" seulement en cas d'erreur. Et "True" sinon (qu'on envoie un mail sans erreur, ou qu'on envoie pas de mail). On pourrait tenter d'être plus malin en renvoyant `None`, mais il faudrait tester tous les envois de mail avec `if send_truc_mail() is False`... et je trouve ça un peu lourd, pour un intérêt peu évident. Qu'en penses-tu ?

Reply:
Tu as raison, je me demandais juste s'il y avait une raison bien particulière au `return False` d'avant.